module/VuFind/src/VuFind/ILS/Driver/Aleph.php

```diff
@@ -1447,7 +1447,7 @@ public function getMyFines($user)
             $checkout = $finesListSort[$key]["checkout"];
             $transactiondate = $finesListSort[$key]["transactiondate"];
             $transactiontype = $finesListSort[$key]["transactiontype"];
-            $balance += $finesListSort[$key]["amount"];
+            $balance = $finesListSort[$key]["amount"];
```

Comment:
$balance is initialized to 0 above in line 1428 and then assigned in line 1437. If we're no longer manipulating that value, is there a reason to use a variable at all? It might make the overall code less confusing to make some changes there.

Reply:
You are right, I will simplify it a bit.